x-pack/plugins/security_solution/server/endpoint/lib/artifacts/migrate_artifacts_to_fleet.ts

```diff
@@ -57,6 +65,12 @@ export const migrateArtifactsToFleet = async (
       }
 
       for (const artifact of artifactList) {
+        if (isCompressed(artifact.attributes)) {
+          artifact.attributes.body = (
+            await inflateAsync(Buffer.from(artifact.attributes.body, 'base64'))
+          ).toString('base64');
+        }
```

Comment:
is there a case where is not compressed? if so, do we have tests to cover it?

Reply:
@paul-tavares correct me if I'm wrong, but if this script is executed just one time it shouldn't be non compressed artifacts right?

Reply:
@academo no, there is no case where an Artifact would be stored in the Saved Object with out being compressed. I think this check is just to be safe, but agree is not technically needed.